.eslintignore

```diff
@@ -0,0 +1,8 @@
+/tests/**

```

Comment:
Is this line of code ignore ESlint for tests? If it does, you should not ignore ESlint for test files.